docker-for-mac/release-notes.md

```diff
@@ -52,6 +52,12 @@ This page contains information about the new features, improvements, known issue
 - The Docker Dashboard displays a tip once a week.
 - Docker Desktop uses iTerm2 to launch the terminal on the container if it is installed. Otherwise, it launches the default Terminal.App. [docker/roadmap#98](https://github.com/docker/roadmap/issues/98)
 - Add experimental support to use the new Apple Virtualization framework (requires macOS Big Sur 11.1 or later)
+- BuildKit is now the default builder for all users, not just new installations. It can still be turned off under **Preferences** > **Docker Engine** with a block like
```

Comment:
We already started using BuildKit as default in version 3.0.0.
Should we move that more down to the 3.0.0 chapter?

Reply:
It was the default for new installations in 2.4 (and it's documented there). I believe it only became the default for existing users in 3.2.